tsconfig.json

```diff
@@ -3,28 +3,18 @@
     "target": "esnext",
     "module": "esnext",
     "lib": [
-      "es6"
+      "esnext",
+      "dom"
     ],
     "strict": true,
     "stripInternal": true,
     "outDir": "dist",
-    "declaration": true,
-    "declarationDir": "dist",
     "allowJs": true,
     "resolveJsonModule": true,
     "moduleResolution": "node",
-    "skipLibCheck": true,
     "paths": {
       "@observablehq/plot": ["./src/index.js"]
     }
   },
-  "files": ["bundle.js"],
-  "include": [
-    "bundle.js",
-    "**/*.ts",
-    "**/*.js",
-  ],
-  "exclude": [
-    "dist"
-  ],
-}
\ No newline at end of file
+  "include": ["src/**/*"]
```

Comment:
Is it right to not include `bundle.js` here? (I'm not entirely clear what that file is for)